chroma-common/blockdevices/blockdevice_zfs.py

```diff
@@ -5,13 +5,35 @@
 import os
 import errno
 import re
-import threading
 import time
+import logging
 
 from collections import defaultdict
-
+from lockfile import LockFile, LockTimeout
 from ..lib import shell
 from blockdevice import BlockDevice
+from ..lib.util import pid_exists
+
+try:
+    from chroma_agent.log import daemon_log as log
+except ImportError:
+    log = logging.getLogger(__name__)
+
+    if not log.handlers:
+        handler = logging.FileHandler('blockdevice_zfs.log')
```

Comment:
we probably should but I don't think we do at the moment, probably not a change for this patch though